src/app/pages/blog/pinned-article/pinned-article.js

```diff
@@ -4,7 +4,7 @@ import useBlogContext from '../blog-context';
 import Section from '~/components/explore-page/section/section';
 import './pinned-article.scss';
 
-export default function PinnedArticle({subhead}) {
+export default function PinnedArticle({subhead=null}) {
```

Comment:
Suggestion: I might be missing if null is needed, but I think you can leave this undefined here